src/Transition/CMakeLists.txt

```diff
@@ -92,7 +96,9 @@ else()  # Windows
   set(STATIC_EXE TRUE)
   if( "${CMAKE_Fortran_COMPILER_ID}" MATCHES "GNU" )
     set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -static")
-    set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
+    if(NOT "Ninja" STREQUAL ${CMAKE_GENERATOR})
+      set(CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} -cpp")
+    endif()
```

Comment:
Interesting.  I wasn't expecting this, but it's ok.

Reply:
see https://gitlab.kitware.com/cmake/cmake/-/issues/17466

Reply:
I have got _no_ problem using the capital F.  Is that handled reliably on Windows (case-insensitive) though?